ext/standard/basic_functions.stub.php

```diff
@@ -179,59 +179,59 @@ function array_change_key_case(array $array, int $case = CASE_LOWER): array {}
 
 function array_unique(array $array, int $flags = SORT_STRING): array {}
 
-function array_intersect_key(array $array1, array $array2, array ...$arrays): array {}
+function array_intersect_key(array $array, array $include, array ...$arrays): array {}
```

Comment:
I'm also not a fan of these `$include`/`$exclude` names

Reply:
Those I want to hold firm on.  array_diff() in particular, it's really not obvious what the heck it does from the name.  By using $exclude/$include, it becomes intrinsically obvious what is going on: return $array, but include/exclude stuff from these other arrays.

Besides, $array2 is the worst option, and I don't know what else I'd call them that isn't obviously inferior to a name that does what it says on the tin.